InformationScripting/src/dataformat/Tuple.cpp

```diff
@@ -28,16 +28,17 @@
 
 namespace InformationScripting {
 
-Tuple::Tuple(std::initializer_list<NamedProperty> initialValues)
-	: values_{initialValues}
+Tuple::Tuple(std::initializer_list<NamedProperty> initialValues, QString tag)
+	: values_{initialValues}, tag_{tag}
 {}
 
-Tuple::Tuple(QList<NamedProperty> initialValues)
-	: values_{initialValues}
+Tuple::Tuple(QList<NamedProperty> initialValues, QString tag)
+	: values_{initialValues}, tag_{tag}
 {}
 
 QString Tuple::tag() const
 {
+	if (!tag_.isNull()) return tag_;
```

Comment:
So this is nice, but do you still think we should support the old behavior (not providing a tag and using the tag of the first part of the tuple)? Are there any drawbacks to removing the old behavior? What are the advantages of keeping this flexibility? I feel like it might cause confusion.